src/main/java/seedu/duke/profile/attributes/Name.java

```diff
@@ -30,6 +31,9 @@ public void setName(String name) {
 
     @Override
     public boolean isValid() {
+        if (name == null || EMPTY_STRING.equals(name)) {
```

Comment:
Should it be name.equals(EMPTY_STRING) instead to make it more appropriate, but trivial stuff ah. Thanks for adding this in 👍

Reply:
Thanks for pointing it out!